rtk_gps_locosys_r1.md

```diff
@@ -0,0 +1,95 @@
+# Freefly Systems RTK GPS
```

Comment:
This is already in docs so remove - see http://docs.px4.io/main/en/gps_compass/rtk_gps_locosys_r1.html